library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -644,7 +644,7 @@ public long addCompletedDownload(String title, String description,
         values.put(Downloads.Impl.COLUMN_MEDIA_SCANNED, (isMediaScannerScannable) ? Request.SCANNABLE_VALUE_YES : Request.SCANNABLE_VALUE_NO);
         values.put(
                 Downloads.Impl.COLUMN_VISIBILITY, (showNotification) ?
-                        Request.VISIBILITY_VISIBLE_NOTIFY_ONLY_COMPLETION : Request.VISIBILITY_HIDDEN);
+                        NotificationVisibility.ONLY_WHEN_COMPLETE : NotificationVisibility.HIDDEN);
```

Comment:
what do you mean? Here is just a renaming + moving out the constants out of the `Request` class. Anyway this method is about adding an already downloaded asset, we will have to revisit it.